docs/en/sql-reference/functions/string-functions.md

```diff
@@ -1415,3 +1415,47 @@ Result:
 │ foo                        │
 └────────────────────────────┘
 ```
+
+## formatQuery
+
+Returns a formatted version of the given SQL query.
+
+**Syntax**
+
+```sql
+formatQuery(query)
+```
+
+**Arguments**
+
+- `query` - The SQL query to be formatted. [String](../../sql-reference/data-types/string.md)
+
+**Returned value**
+
+- The formatted query. [String](../../sql-reference/data-types/string.md).
+
+**Example**
+
+```sql
+SELECT formatQuery('select 1;');
+```
+
+Result:
+
+```result
+┌─formatQuery('select 1;')─┐
+│ SELECT 1                 │
+└──────────────────────────┘
+```
+
+```sql
+SELECT formatQuery('select 1;') == formatQuery('SeLecT 1');
```

Comment:
I would probably make the example more straightforward, for example, `formatQuery('SeLecT 1');` to actually see what this function returns